ethereum/migrations/4_dump_address.js

```diff
@@ -0,0 +1,31 @@
+require("dotenv").config();
```

Comment:
re filename: 4_ is used, can we make this 6_ or keep it as 4_ and bump the others up by one?

Reply:
fixed. Renamed to 99_dump_address.js